BUILD.md

```diff
@@ -0,0 +1,39 @@
+Environment
+This documentation is intended for Ubuntu 20.4
+
+### Docker
+
+To begin with, you should install Docker. If you haven't installed it yet, please run the following command:
+
+`bash docker_install.sh`
+
+### Verifier
+
+Build the Docker image for the verifier:
+
+`docker build -t acs:0.1 verifier/
+`docker run -d -p 2323:2323 -it acs:0.1 bash /root/startup.sh >/dev/null 2>&1`
+
```

Comment:
The end of `docker build` command lacks of `. In addition, please use the following format.
```sh
docker build -t acs:0.1 verifier/
docker run -itd -p 2323:2323 acs:0.1 bash /root/startup.sh >/dev/null 2>&1`
```